en/appendices/3-4-migration-guide.rst

```diff
@@ -105,9 +105,6 @@ Other Deprecations
 * ``Router::parse()`` is deprecated. ``Router::parseRequest()`` should be used
   instead as it accepts a request and gives more control/flexibility in handling
   incoming requests.
-* ``Route::parse()`` is deprecated. ``Route::parseRequest()`` should be used
```

Comment:
Why would you remove this ? 
`Router::parse()` and `Route::parse ()` are not the same and are both deprecated.